client/src/pages/Questions/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+import Tabs from './components/Tabs'
+
+export default function QuestionsPage(props) {
+
+  return (
+    <Tabs />
```

Comment:
There is no need for passing the component here and pass it to the app component ??
what do you thing ?

Reply:
Why to pass it to app component?  
only QuestionsPage

Reply:
I think because we are only rendering Tabs in this page, we could actually render Tabs in app.js and be able to delete the questionPage. I think that is the suggestion?

Reply:
@dubhcait  Abdullah suggest to remove Tabs component and puts its code inside QuestionsPage, I think this is better!